packages/common/amplify_db_common/test/main_test.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(Jordan-Nelson): Run tests on web. This will require loading sqlite3.wasm
+@TestOn('vm')
+
+import 'package:amplify_db_common/amplify_db_common.dart';
+import 'package:drift/drift.dart';
+import 'package:test/test.dart';
+
+// ignore: avoid_relative_lib_imports
+import '../example/lib/database.dart';
+
+void main() {
```

Comment:
Should we rename this class to use io/vm/flutter to indicate it's for non web cases?

Reply:
main_test vs connect.vm - seems weird main indicates non web for unit testing context and vm indicates non web for split platform code cases

Reply:
Ah ok thanks I misunderstood the purpose of the class then.